### PR TITLE
Beautify `cap_*` methods signatures

### PR DESCRIPTION
**Pull request by eugulixes**

### CODE REVIEW

File: gits/terminal.py [hunk 1]
```diff
@@ -323,68 +308,59 @@ def _cap_cvvis(self):
...
-        """Deletes the specified number of lines. """
-        if mo:
-            p1 = int(mo.group(1))
+        self._cap_set_color(2)
 
+    def _cap_dl(self, n):
+        """Deletes ``n`` number of lines. """
         if self._top_most <= self._cur_y <= self._bottom_most:
-            for i in range(p1):
+            for i in range(n):
```
Comment (dshil) [inline gits/terminal.py]: Can we omit unused loop variable `i`?

File: gits/terminal.py [hunk 2]
```diff
@@ -323,68 +308,59 @@ def _cap_cvvis(self):
...
 
-    def _cap_home(self, l=None):
+    def _cap_home(self):
         """Moves the cursor to the home position. """
-        if l is None:
-            l = [1, 1]
-        self._cur_x = min(self._cols, l[1]) - 1
-        self._cur_y = min(self._rows, l[0]) - 1
+        self._cur_x = min(self._cols, 1) - 1
+        self._cur_y = min(self._rows, 1) - 1
```
Comment (dshil) [inline gits/terminal.py:363]: Let's simplify
```python
self._cur_x = min(self._cols, 1) - 1
self._cur_y = min(self._rows, 1) - 1
```
to
```python
self._cur_x = self._cur_y = 0
```

File: gits/test/helper.py [hunk 3]
```diff
@@ -453,14 +448,16 @@ def _check_cap_home(self, pos):
         # Restore the terminal to the sane modes.
         term._cap_rs1()
 
-    def _check_cap_hpa(self, mo):
-        """A helper that checks the `_cap_hpa` method. """
+    def _check_cap_hpa(self, x):
+        """A helper that checks the `_cap_hpa` method.
+
+
```
Comment (dshil) [inline gits/test/helper.py]: Do we need extra blank lines?

File: gits/test/helper.py [hunk 4]
```diff
@@ -573,9 +570,11 @@ def _check_cap_ri(self, s, pos):
         # Reset the terminal to sane modes.
         term._cap_rs1()
 
-    def _check_cap_vpa(self, mo=None):
-        """A helper that checks the `_cap_vpa` method. """
+    def _check_cap_vpa(self, y):
+        """A helper that checks the `_cap_vpa` method.
+
+
```
Comment (dshil) [inline gits/test/helper.py]: Extra blank lines

File: gits/test/helper.py (hunk 3 above)
Comment (eugulixes) [reply gits/test/helper.py]: I wanted to describe `x` but forgot.

File: gits/terminal.py (hunk 2 above)
Comment (eugulixes) [reply gits/terminal.py:363]: I didn't chage anything. Let's do it in a different PR.

**Review by dshil — APPROVED** [review]
LGTM

**Pull request merged**